src/solvers/dg_curved/containers_2d.jl

```diff
@@ -0,0 +1,91 @@
+function init_elements!(elements, mesh::CurvedMesh{2}, basis::LobattoLegendreBasis{T, NNODES}) where {T, NNODES}
+  @unpack faces = mesh
+  @unpack node_coordinates, left_neighbors, metric_terms, inverse_jacobian = elements
+
+  linear_indices = LinearIndices(size(mesh))
+
+  # Calculate inverse Jacobian and node coordinates
+  for cell_x in 1:size(mesh, 1), cell_y in 1:size(mesh, 2)
```

Comment:
```suggestion
  for cell_y in 1:size(mesh, 2), cell_x in 1:size(mesh, 1)
```
Julia uses row-major memory layout. Thus, this iteration order follows the memory layout.